CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ git commit -m "test: Add test suite for feature x"
 
 Further details on `conventional commits` can be found here: <https://www.conventionalcommits.org/en/v1.0.0/>
 
-When merging a branch to `develop`, PRs should be squashed into one conventional commit by selecting the `Squash and merge` option. This ensures Release notes are useful and readable when releases are created.
+When merging a branch to `master`, PRs should be squashed into one conventional commit by selecting the `Squash and merge` option. This ensures Release notes are useful and readable when releases are created.
```

Comment:
We do not want to squash anymore, only `Merge commit`. Remove this rule and I will update the docs with more explanation later.